src/m365/planner/commands/task/task-add.ts

```diff
@@ -293,7 +329,10 @@ class PlannerTaskAddCommand extends GraphCommand {
       { option: '--percentComplete [percentComplete]' },
       { option: '--assignedToUserIds [assignedToUserIds]' },
       { option: '--assignedToUserNames [assignedToUserNames]' },
+      { option: '--assigneePriority [assigneePriority]' },
       { option: '--description [description]' },
+      { option: '--appliedCategories [appliedCategories]' },
+      { option: '--previewType [previewType]' },
```

Comment:
Since we're working with a predefined list of types, let's add them here to autocomplete for easier use